repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -2054,13 +2042,13 @@ public List appendArrayValue(AttributeMutationContext ctx, EntityMutationContext
             newElements = (List) newElements.stream().distinct().collect(Collectors.toList());
         }
 
+
         for (int index = 0; index < newElements.size(); index++) {
-            AtlasEdge               existingEdge = (isSoftReference) ? null : getEdgeAt(currentElements, index, elementType);
             AttributeMutationContext arrCtx      = new AttributeMutationContext(ctx.getOp(), ctx.getReferringVertex(), ctx.getAttribute(), newElements.get(index),
-                    ctx.getVertexProperty(), elementType, existingEdge);
+                    ctx.getVertexProperty(), elementType, null);
```

Comment:
Donot add `null` argument in the constructor, you can probably overload the constructor ?

Reply:
done